path.red

```diff
@@ -67,9 +67,8 @@ let symm/unit
 
 let trans/filler
   (A : type)
-  (x : A)
   (p : Line A)
-  (q : Path A (p 1) x)
+  (q : [i] A with i=0 ⇒ p 1 end)
```

Comment:
ah, interesting! I forgot that of course one can have an extension *pretype* without a valid system. nicely done.